.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_VERSION=$npm_package_version
+REACT_APP_API_URL=http://localhost:4000/api
```

Comment:
Did this file has to be uploaded or added to the `.gitignore`?

Reply:
Maybe, you are right, with respect to `.gitignore`, it happens that it was added as `.env` instead of `client/.env`. I think that since it is on the client's side, we will never put important things on the application, but to avoid surprises I will change it and put a boilerplate on .env.